docs/public/_redirects

```diff
@@ -139,3 +136,11 @@ https://v3-9-0.material-ui.com/* https://v3.material-ui.com/:splat 301!
 /v0.3.2 https://v0.material-ui.com/v0.3.2
 /v0.3.1 https://v0.material-ui.com/v0.3.1
 /v0.3.0 https://v0.material-ui.com/v0.3.0
+
+# Proxies
+/store/* https://material-ui-store.netlify.app/:splat 200
+/store-staging/* https://master--material-ui-store.netlify.app/:splat 200
+/components/data-grid/pagination/ https://material-ui-x.netlify.app/components/data-grid/pagination/ 200
```

Comment:
will use

```
/components/data-grid/* https://material-ui-x.netlify.app/components/data-grid/:splat 200
```

when we have more pages, for now, it's only mean to show that it can work